vpr/src/draw/breakpoint.cpp

```diff
@@ -0,0 +1,107 @@
+#include "breakpoint.h"
+#include "draw_global.h"
+
+#include <iostream>
+
+//global variables
+int tempCount = 0;
+current_information current_info_b;
```

Comment:
the issue with defining it in the breakpoint class is that it needs to be accessed in the expr_eval class as well which would make things more complicated as expr_eval is included in the breakpoint class.

Reply:
I think you're saying that you're trying to avoid a circular dependency between headers (breakpoint.h #includes vtr_eval.h and vtr_eval.h #includes breakpoint.h)?
Some general ideas for avoiding that are here: https://stackoverflow.com/questions/4816698/avoiding-circular-dependencies-of-header-files
Probably the most general one is to factor the header files into multiple smaller ones, with some just giving limited interfaces (used by more distant code) and some giving more detailed interfaces (used by other code, perhaps more tightly joined).